pkg/dtclient/dynatrace_client.go

```diff
@@ -64,25 +64,35 @@ func (dc *dynatraceClient) makeRequest(url string, tokenType tokenType) (*http.R
 	return dc.httpClient.Do(req)
 }
 
-func (dc *dynatraceClient) getServerResponseData(response *http.Response) ([]byte, error) {
+func (dc *dynatraceClient) getServerResponseData(response *http.Response) ([]byte, *serverError, error) {
```

Comment:
You can use Go 1.13 errors to manage specific types of errors. I'm currently working on some similar changes for this, so we can leave it as it's now.